modality-probe-capi/README.md

```diff
@@ -419,26 +419,26 @@ something like this:
 *   |   CONSUMER_STARTED @ CONSUMER_PROBE (1:30020207:0:3)
 |   |       description: "Measurement consumer thread started"
 |   |       tags: consumer
-|   |       source: "rust-example/src/main.rs#L141"
-|   |       probe_tags: rust-example, measurement, consumer
-|   |       probe source: "rust-example/src/main.rs#L129"
+|   |       source: "c-example/src/main.rs#L196"
+|   |       probe tags: c-example, measurement, consumer
+|   |       probe source: "c-example/src/main.rs#L186"
 |   |       component: example-component
 |   |
 |   *   PRODUCER_STARTED @ PRODUCER_PROBE (1:837318897:0:3)
 |   |       description: "Measurement producer thread started"
 |   |       tags: producer
-|   |       source: "rust-example/src/main.rs#L77"
-|   |       probe_tags: rust-example, measurement, producer
-|   |       probe source: "rust-example/src/main.rs#L65"
+|   |       source: "c-example/src/main.rs#L102"
+|   |       probe tags: c-example, measurement, producer
+|   |       probe source: "c-example/src/main.rs#L92"
 |   |       component: example-component
 |   |
 |   *   PRODUCER_MEASUREMENT_SAMPLED @ PRODUCER_PROBE (1:837318897:0:4)
 |   |       description: "Measurement producer sampled a value for transmission"
 |   |       payload: 1
 |   |       tags: producer, measurement sample
-|   |       source: "rust-example/src/main.rs#L91"
-|   |       probe_tags: rust-example, measurement, producer
-|   |       probe source: "rust-example/src/main.rs#L65"
+|   |       source: "c-example/src/main.rs#L144"
+|   |       probe_tags: c-example, measurement, producer
```

Comment:
`probe_tags` still has the underscore here

Reply:
:+1: